elm/src/Test/Runner/Node.elm

```diff
@@ -309,11 +309,52 @@ init { processes, globs, paths, fuzzRuns, initialSeed, report, runners } _ =
     ( model, Cmd.none )
 
 
+{-| The implementation of this function will be replaced in the generated JS
+with a version that returns `Just value` if `value` is a `Test`, otherwise `Nothing`.
+
+If you rename or change this function you also need to update the regex that looks for it.
+
+-}
+check : a -> Maybe Test
+check value =
+    Debug.todo "The regex for replacing this Debug.todo with some real code must have failed since you see this message!\n\nPlease report this bug: https://github.com/rtfeldman/node-test-runner/issues/new\n"
```

Comment:
You are probably fed up with my never ending suggestions about this function so please feel free to ignore me. This might be the best of both worlds though (crashes if our regex fails and should allow a simply regex):

```suggestion
check : a -> Maybe Test
check =
    checkHelperReplaceMe___
    
 
checkHelperReplaceMe___ : a -> b
checkHelperReplaceMe___ _=
    Debug.todo "The regex for replacing this Debug.todo with some real code must have failed since you see this message!\n\nPlease report this bug: https://github.com/rtfeldman/node-test-runner/issues/new\n"
```

Reply:
Great idea! :+1: